code/game/machinery/telecomms/logbrowser.dm

```diff
@@ -75,37 +76,34 @@
 						var/race			   // The actual race of the mob
 						var/language = "Human" // MMIs, pAIs, Cyborgs and humans all speak Human
 						var/mobtype = C.parameters["mobtype"]
-						var/mob/M = new mobtype
 
-						if(ishuman(M) || isbrain(M))
-							var/mob/living/carbon/human/H = M
-							race = "[H.species.name]"
+						if(ispathhuman(mobtype) || ispathbrain(mobtype))
+							race = "Human" // The initializing thing wouldn't work anyways since it just kept the path, and kept no species data
 
 
-						else if(issmall(M))
+						else if(ispath(mobtype, /mob/living/carbon/human/monkey)) // I am aware this will only work for always-monkeys, and not
```

Comment:
We're working off of paths here, which contain no useful information if the person is actually a monkey or not. This block will never get called.